src/tags-input/tags-input.css

```
@@ -61,7 +61,7 @@
 
   caret-color: var(--ring-main-color);
 
-  @nest [dir=rtl] & {
+  [dir="rtl"] & {
```

Oops, it should be:
```suggestion
  [dir=rtl] & {
```
I will fix it.